content/conditions/warts/main-content.md

```diff
@@ -1,13 +1,15 @@
 Warts don’t cause you any harm but some people find them itchy, painful or embarrassing.
-Verrucas are more likely to be painful - like standing on a needle.\n\nYou can treat warts if they’re bothering you,
-they keep coming back or they’re painful.
+Verrucas are more likely to be painful - like standing on a needle.
```

Comment:
AFAIK Em dash requires no space around it. Should it be en dash instead? And if so, we should be consistent in its usage. Anyway, I think this is for content designers to decide.